packages/notifi-dataplane/lib/types/FusionMessage.ts

```diff
@@ -1,33 +1,12 @@
 import { Types } from 'notifi-graphql/dist';
 
-// If payload type undefined, fallback to default CommunityManagerJsonPayload
-export interface FusionMessage<T extends object = CommunityManagerJsonPayload> {
+export interface FusionMessage {
   eventTypeId: string;
-  variablesJson: VariablesJsonPayload<T>;
+  variablesJson: object;
   specificWallets?: ReadonlyArray<
     Readonly<{
       walletPublicKey: string;
       walletBlockchain: Types.WalletBlockchain;
     }>
   >;
 }
```

Comment:
@hans-notifi , Just realized you are right, there is a mistaking forcing the Generic in `FusionMessage` as `CommunityManagerJsonPayload`. 
We will be able to fix it by: 

```suggestion
export interface FusionMessage<T extends object = object> {
  eventTypeId: string;
  variablesJson: VariablesJsonPayload<
    T extends CommunityManagerJsonPayload ? CommunityManagerJsonPayload : object
  >;
  specificWallets?: ReadonlyArray<
    Readonly<{
      walletPublicKey: string;
      walletBlockchain: Types.WalletBlockchain;
    }>
  >;
}
```
Then it will allow putting any shape of object ex.